src/xtopcom/xtxpool_service_v2/src/xtxpool_service.cpp

```diff
@@ -414,7 +419,7 @@ void xtxpool_service::drop_msg(vnetwork::xmessage_t const & message, std::string
 }
 
 void xtxpool_service::on_message_receipt(vnetwork::xvnode_address_t const & sender, vnetwork::xmessage_t const & message) {
-    if (!is_running()) {
+    if (status() == enum_txpool_service_status_not_run) {

```

Comment:
no need to return for enum_txpool_service_status_faded?

Reply:
should process messages received,or else would lose receipts.